dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -930,16 +930,23 @@ public void inheritCollectionDefaultPolicies(Context context, Item item, Collect
     @Override
     public void adjustBundleBitstreamPolicies(Context context, Item item, Collection collection)
         throws SQLException, AuthorizeException {
-        List<ResourcePolicy> defaultCollectionPolicies = authorizeService
+        // Bundles should inherit from DEFAULT_ITEM_READ so that if the item is readable, the files
+        // can be listed (even if they are themselves not readable as per DEFAULT_BITSTREAM_READ or other
+        // policies or embargos applied
+        List<ResourcePolicy> defaultCollectionBundlePolicies = authorizeService
+                .getPoliciesActionFilter(context, collection, Constants.DEFAULT_ITEM_READ);
+        // Bitstreams should inherit from DEFAULT_BITSTREAM_READ
+        List<ResourcePolicy> defaultCollectionBitstreamPolicies = authorizeService
```

Comment:
@kshepherd : Since, it appears all tests still succeed with this change... could you add a *new test* which verifies this behavior?  Basically a test which creates an Item which inherits both `DEFAULT_ITEM_READ` and `DEFAULT_BITSTREAM_READ` from its Collection and verify that the Bundle inherits the former, while the Bitstream inherits the latter.  (It's possible you could modify an existing test to simply add in the Bundle check... I suspect we just don't have any tests to verify how a Bundle inherits permissions.)

Reply:
Added this test for you in https://github.com/DSpace/DSpace/pull/8901/commits/804fce1c99419b29e745119460f3c351a8dc1a18